Module 2.4

N1ckky opened the pull request:
Module webdriver for review

N1ckky committed:
Added first task in 2.4

Upload first task: "I Can Win"

N1ckky committed:
Remove some lines

Remove all commited lines from code

N1ckky committed:
Upload second task in 2.4

Upload "Bring It On" task in module 2.4

N1ckky committed:
upload third task in module 2.4

Upload task "HurtMePlenty" in module with webdriver

N1ckky committed:
upload fourth task in module 2.4

Upload completed task "Hardcore" in module 2.4.
All tasks from webdriver is finished and prepeared for review

AndreiValnianka commented on Nick_Maven/webdriver/Hardcore/POModel/CloudGoogleHomePage.java:
```diff
@@ -0,0 +1,38 @@
+
+
+
+    @FindBy(xpath = "//*[@class='button button-white devsite-suggest-all-results']")
```
This is 3 classes:
button
button-white
devsite-suggest-all-results
If all of them need for locator use contains().

AndreiValnianka commented on Nick_Maven/webdriver/Hardcore/POModel/CloudGooglePlatformPricingCalculatorPage.java:
```diff
@@ -0,0 +1,240 @@
+
+    private static final String JS_CLICK = "arguments[0].click();";
+    private static final String TAG_IFRAME = "iframe";
+    private static final String NUMBER_OF_INSTANCES = "4";
```
move constants to the top

AndreiValnianka commented on Nick_Maven/webdriver/Hardcore/POModel/CloudGooglePlatformPricingCalculatorPage.java:
```diff
@@ -0,0 +1,240 @@
+//        System.out.println("DEBUG check condition " + checkFlag);
+//        return checkFlag;
+//    }
+
```
delete unused code

AndreiValnianka commented on Nick_Maven/webdriver/Hardcore/Test/Hardcore.java:
```diff
@@ -0,0 +1,73 @@
+        driver = new ChromeDriver();
+    }
+
+    @Test(description = "JIRA binding can be here")
```
strange description for this test

AndreiValnianka commented on Nick_Maven/webdriver/HurtMePlenty/Test/HurtMePlentyTest.java:
```diff
@@ -0,0 +1,47 @@
+        driver = new ChromeDriver();
+    }
+
+    @Test(description = "JIRA binding can be here")
```
strange description for this test

AndreiValnianka reviewed (changes requested):
1. Adjust locators
2. Delete unused code
3. Add an appropriate description for test methods
4. Delete extra strings

N1ckky committed:
Error correction

Error correction after review.
Deleted some unused lines;
added description to tests;
adjustion locators
